README.md

```diff
@@ -313,3 +314,17 @@ e.g. `helm --namespace mojaloop del dev`
 ## Monitoring Mojaloop
 
 Refer to [Monitoring Documentation](./monitoring/README.md)
+
+## Batch Processing
+
+To enable batch processing in your system, please follow the steps below:
```

Comment:
What about the changes needed to prepare Handler config to change the topic mapping?

Reply:
Hi @mdebarros that is already handled by the same config parameter. Please refer the file centralledger/chart-handler-transfer-prepare/configs/default.json

Reply:
Thanks, can you please add some kind of description as to what its doing, i.e. overriding the topic mapping for position-prepare (pls confirm this) events to be re-directed to the batch processing topic, and additional information that as long as the existing position handler is running...all pre-batch deployment will be processed as normal in parallel while new position-prepare events will be processed by the batch processor.